commands/utils/params.go

```diff
@@ -290,7 +292,16 @@ func extractGitParamsFromEnv() (*Git, error) {
 	gitParams.Branches = append(gitParams.Branches, branch)
 	if pullRequestIDString := getTrimmedEnv(GitPullRequestIDEnv); pullRequestIDString != "" {
 		gitParams.PullRequestID, err = strconv.Atoi(pullRequestIDString)
+		if err != nil {
+			return nil, err
+		}
+	}
+	// Non-mandatory git aggregate pull requests feature
+	aggregatePullRequestsEnv, err := getBoolEnv(GitAggregatePullRequestsEnv, false)
```

Comment:
aggregatePullRequestsEnv --> aggregateFixesEnv
(let's change all functions and related names throughout the code accordingly).